Common/include/interface_interpolation/CSlidingMesh.hpp

```diff
@@ -106,6 +106,14 @@ class CSlidingMesh final : public CInterpolator {
   static su2double ComputeIntersectionArea(const su2double* P1, const su2double* P2, const su2double* P3,
                                            const su2double* Q1, const su2double* Q2, const su2double* Q3);
 
+  /*!
+   * \brief For 3-Dimensional grids, compute area between of one triangle
+   * \param[in] P1 - first  point of triangle
+   * \param[in] P2 - second point of triangle
+   * \param[in] P3 - third  point of triangle
+   */
+  static su2double ComputeTriangleArea(const su2double* P1, const su2double* P2, const su2double* P3);
```

Comment:
Since it's just one line it's better to define it inline. We may already have something similar in GeometryToolbox, and if not you can add this function to it.